erizo_controller/erizoJS/models/SessionDescription.js

```diff
@@ -68,7 +68,9 @@ function getMediaInfoFromDescription(info, sdp, mediaType) {
 
   let ice = info.getICECredentials(mediaType);
   if (ice) {
-    media.setICE(new ICEInfo(ice[0], ice[1]));
+    const thisIceInfo = new ICEInfo(ice[0], ice[1]);
+    thisIceInfo.setEndOfCandidates('end-of-candidates'); 
```

Comment:
does this break something in Chrome/Firefox?

Reply:
this is bulletproof

Reply:
This is breaking things in chrome, local and remote stream turns black after merging this change